Source/Libraries/openXDA.PQI/PQIWSClient.cs

```diff
@@ -306,8 +306,10 @@ private static string BuildURL(params string[] parts)
 
                 if (combinedPath.Length == 0)
                     combinedPath = path;
+                else if (path.StartsWith(Separator) && combinedPath.EndsWith(Separator))
+                    combinedPath += path.TrimStart(Separator.ToCharArray());
                 else if (path.StartsWith(Separator))
-                    combinedPath = path;
+                    combinedPath += path;
```

Comment:
Although I do agree that this behavior is nonsensical, and I don't think that changing it will cause any bugs, I do think it's worth noting that this was intentionally done so it behaves similar to `Path.Combine()`. It may be worth getting out of the habit of allowing `/` at the beginning of relative paths.

If you insist, here's a better way to remove the separator from the start of `path`.

```diff
-combinedPath += path.TrimStart(Separator.ToCharArray());
+combinedPath += path.Substring(Separator.Length);
```

Reply:
Oh, I see the problem now. I guess the PQI web service returns paths like `/Area?facilityAuditID=1` which are meant to indicate an absolute path with the same base URL as the page that queried them, and we're treating them like relative paths here. I think I can live with this fix.